gson/src/main/java/com/google/gson/JsonObject.java

```diff
@@ -155,6 +155,15 @@ public int size() {
     return members.size();
   }
 
+  /**
+   * Returns true if the number of key/value pairs in the object is zero.
+   *
+   * @return true if the number of key/value pairs in the object isgit sd zero.
```

Comment:
```suggestion
   * @return true if the number of key/value pairs in the object is zero.
```
Or alternatively maybe you could also omit the `@return` since it does not add much value, see also [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment), which implicitly shows that `@return` can be omitted (if I understand it correctly).

Reply:
Done.